src/main/java/snowy/Snowy.java

```diff
@@ -0,0 +1,62 @@
+package snowy;
+
+import snowy.common.Command;
+import snowy.common.CommandResult;
+import snowy.data.SnowyException;
+import snowy.storage.Storage;
+import snowy.tasklist.TaskList;
+import snowy.ui.TextUi;
+import snowy.parser.Parser;
+
+public class Snowy {
+    private static final String FILE_PATH = "../ip/src/main/data";
+    private static final String FILENAME = "snowy.txt";
```

Comment:
I like how the names of constant variables appropriately follow the coding standard and how this is consistent throughout your code.